src/main/java/com/jfrog/ide/idea/ui/jcef/message/MessageType.java

```diff
@@ -0,0 +1,16 @@
+package com.jfrog.ide.idea.ui.jcef.message;
+
+public enum MessageType {
+    SetEmitter("SET_EMITTER"),
```

Comment:
```suggestion
    SET_EMITTER, SHOW_PAGE
```
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html
I think this will be easier to read and follows Java conventions :)